setup.cfg

```diff
@@ -15,13 +15,13 @@ classifier =
     Programming Language :: Python
 [files]
 packages =
-    tools
+    oca
```

Comment:
Im agree with @pedrobaeza 
Maybe, oca_mt
To work in a future next package with oca_* (maybe oca_mqt to check lints)